lib/api.js

```diff
@@ -70,6 +70,106 @@ export async function getallProjectsWithSlug() {
   return data?.allProjects
 }
 
+export async function getAllProjectsByAuthor(id) {
+  const data = fetchAPI(`
+    query getProjectsByAuthor($id: ItemId) {
+      allProjects(filter: {author: {eq: $id}}) {
+        id
+        date
+        title
+        slug
+        coverImage {
+          responsiveImage(imgixParams: {fm: jpg, fit: crop, w: 2000, h: 1000 }) {
+            ...responsiveImageFragment
+          }
+        }
+        excerpt
+        author {
+          slug
+          picture {
+            url
+          }
+        }
+        tags {
+          name
+        }
+      }
+    }
+    ${responsiveImageFragment}
+    `,
+    {
+      variables: {
+        id
+      }
+    }
+  )
+  return data;
+}
+
+export async function getAllProjectsByTag(id) {
+  const data = fetchAPI(`
+    query getProjectsByTag($tagId: [ItemId]) {
```

Comment:
I think this needs to be $id ...

Page designs look great, but its pulling all projects, not executing the query -- see https://datocms-next-js-blog-demo-2616-git-more-design-page.philipkobernik.vercel.app/tags/sustainability